build-system/tasks/extension-generator/index.js

```diff
@@ -274,10 +274,13 @@ module.exports = {
   insertExtensionBundlesConfig,
 };
 
+/* eslint "google-camelcase/google-camelcase": 0 */
+
 makeExtension.description = 'Create an extension skeleton';
 makeExtension.flags = {
   name: '  The name of the extension. Preferable prefixed with `amp-*`',
   bento: '  Generate a Bento component',
+  no_css: '  Generates extension without CSS boilerplate; --bento only',
```

Comment:
I think the linter won't yell at you if you use quotes.

```suggestion
makeExtension.description = 'Create an extension skeleton';
makeExtension.flags = {
  name: '  The name of the extension. Preferable prefixed with `amp-*`',
  bento: '  Generate a Bento component',
  'no_css': '  Generates extension without CSS boilerplate; --bento only',
```